lighthouse-core/test/gather/fake-driver.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     return Promise.resolve();
   },
   gotoURL() {
-    return Promise.resolve('https://example.com');
+    return Promise.resolve('https://www.reddit.com/r/nba');
```

Comment:
that's the correct URL for the `../fixtures/traces/progressive-app.json` returned by fake-driver